serverless-test-project/serverless-v2.yml

```diff
@@ -98,6 +98,7 @@ functions:
             enabled: false
     events:
       - eventBridge:
+          useCloudFormation: true
```

Comment:
Do you know why this property is part of this feature?

Reply:
Yes, because EventBridge rule is created using CloudFormation instead of a Custom Resource. This requires for Serverless Framework  2.27.0. We have to set it as true otherwise it was outputting error.
https://www.serverless.com/framework/docs/providers/aws/events/event-bridge#:~:text=Try%20it%20Now-,EventBridge%20Event,-The%20EventBridge%20makes